lib/lookup.json

```diff
@@ -1,315 +1,396 @@
+    "flaky": ["ppc", "darwin"],
+    "maintainers": ["isaacs", "calvinmetcalf"]
   },
   "ftp": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "mscdex"
   },
   "split2": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "mcollina"
   },
   "throughv": {
     "prefix": "v",
-    "stripAnsi": true
+    "stripAnsi": true,
+    "maintainers": "mcollina"
   },
   "duplexer2": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": ["shinn", "deoxxa"]
   },
   "bl": {
     "prefix": "v",
     "stripAnsi": true,
-    "expectFail": "fips"
+    "expectFail": "fips",
+    "maintainers": "rvagg"
   },
   "binary-split": {
     "prefix": "v",
-    "flaky": "win32"
+    "flaky": "win32",
+    "maintainers": ["maxogden", "mourner"]
   },
   "spdy": {
     "prefix": "v",
     "flaky": ["aix", "sles"],
-    "skip": "win32"
+    "skip": "win32",
+    "maintainers": "indutny"
   },
   "dicer": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "mscdex"
   },
   "spdy-transport": {
     "prefix": "v",
     "skip": "win32",
-    "flaky": "aix"
+    "flaky": "aix",
+    "maintainers": "indutny"
   },
   "sax": {
     "prefix": "v",
-    "expectFail": "fips"
+    "expectFail": "fips",
+    "maintainers": "isaacs"
   },
   "duplexify": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "mafintosh"
   },
   "pumpify": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "mafintosh"
   },
   "from2": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": ["mafintosh", "hughsk"]
   },
   "flush-write-stream": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "mafintosh"
   },
   "jsonstream": {
     "prefix": "v",
-    "skip": "win32"
+    "skip": "win32",
+    "maintainers": "dominictarr"
   },
   "csv-parser": {
     "prefix": "v",
-    "flaky": "win32"
+    "flaky": "win32",
+    "maintainers": "mafintosh"
   },
   "async": {
     "prefix": "v",
-    "flaky": true
+    "flaky": true,
+    "maintainers": ["aearly", "megawac"]
   },
   "bluebird": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": "petkaantonov"
   },
   "mkdirp": {
     "master": true,
     "flaky": ["v6", "v7"],
-    "skip": ["v8"]
+    "skip": ["v8"],
+    "maintainers": "substack"
   },
   "yeoman-generator": {
     "prefix": "v",
     "flaky": ["ppc", "win32", "rhel"],
-    "expectFail": "fips"
+    "expectFail": "fips",
+    "maintainers": ["SBoudrias", "sindresorhus"]
   },
   "minimist": {
+    "maintainers": "substack"
   },
   "cheerio": {
-    "skip": "win32"
+    "skip": "win32",
+    "maintainers": ["matthewmueller", "jugglinmike"]
   },
   "uuid": {
-    "prefix": "v"
+    "prefix": "v",
+    "maintainers": ["ctavan", "broofa"]
   },
   "winston": {
-    "flaky": ["win32", "ppc"]
+    "flaky": ["win32", "ppc"],
+    "maintainers": "indexzero"
   },
   "yargs": {
     "prefix": "v",
     "flaky": "aix",
-    "expectFail": "fips"
+    "expectFail": "fips",
+    "maintainers": "bcoe"
```

Comment:
you can add me if you want (not a maintainer but would like the notifications :))